client/src/reducers/controls.js

```diff
@@ -138,6 +138,11 @@ const Controls = (
     /*******************************
              User Events
      *******************************/
+    case "change graph interaction mode":
```

Comment:
You need to configure undo/redo handling whenever you change actions (either add/del an action, or change its semantics). 

If you simply want this to push a history undo point, as I suspect is the case, add this action type to the `saveOnActions` Set in undoableConfig.js.   If you want something more complex, let me know.